samples/AzCopyCore/AzCopyCore/Program.cs

```diff
@@ -76,7 +79,9 @@ static void TransferDirectoryToStorage(ReadOnlySpan<char> localDirectory, ReadOn
         byte[] keyBytes;
         if (options.Contains("/DestKey:"))
         {
-            keyBytes = options["/DestKey:"].ComputeKeyBytes();
+            ReadOnlySpan<char> key = options["/DestKey:"];
+            var str = key.ToString();
+            keyBytes = key.ComputeKeyBytes();
```

Comment:
Why is this change necessary?

Reply:
leftover from debugging. I removed now. thanks.

Reply:
FYI, I just confirmed, debugging works as expected and shows the string. So, hopefully, these types of changes won't be necessary anymore.

![image](https://user-images.githubusercontent.com/6527137/37736030-fff95b28-2d0c-11e8-8a15-ad890c57e0b8.png)